Src/AutoFixtureUnitTest/RandomNumericSequenceGeneratorTest.cs

```diff
@@ -108,9 +121,9 @@ public void LimitsDoesNotMatchParamsArray()
             // Fixture setup
             var unexpectedLimits = new long[]
             { 
-                Byte.MaxValue, 
-                Int16.MaxValue, 
-                Int32.MaxValue
+                (byte)10,
+                (Int16)50,
+                (Int32)20,

```

Comment:
OK, thank you for explaining this :+1:

 #409 takes care of that unit test (so we don't have to, in this PR).

I think, if we can rollback this change, we might avoid unnecessary merge conflicts :smiley: